### PR TITLE
[Cache] Updates cache purge

### PR DESCRIPTION
**Pull request by angelampcosta**
### Summary

Updates to cache purge docs.

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/cache/how-to/purge-cache/purge-by-tags.mdx [hunk 1]
```diff
@@ -6,14 +6,14 @@ sidebar:
...
+Cache-tag purging makes multi-file purging easier because you can instantly bulk purge by adding cache-tags to your assets, such as webpages, image files, and more. Note that Tag, Hostname and Prefix purges are only available for Enterprise customers.
 
 ## General workflow for cache-tags
 
 1. Add tags to the `Cache-Tag` HTTP response header from your origin web server for your web content, such as pages, static assets, etc.
 2. [Ensure your web traffic is proxied](/dns/manage-dns-records/reference/proxied-dns-records/) through Cloudflare.
 3. Cloudflare associates the tags in the `Cache-Tag` HTTP header with the content being cached.
-4. Use specific cache-tags to purge your Cloudflare CDN cache of all content containing that cache-tag from your dashboard or [using our API](/api/operations/zone-purge).
+4. Use specific cache-tags to immediately purge your Cloudflare CDN cache of all content containing that cache-tag from your dashboard or [using our API](/api/operations/zone-purge).
```
Comment (tkornhammar) [inline src/content/docs/cache/how-to/purge-cache/purge-by-tags.mdx]: Use word "instantly".

File: src/content/docs/cache/advanced-configuration/cache-reserve.mdx [hunk 2]
```diff
@@ -121,7 +121,7 @@ Class B operations are performed when data needs to be fetched from Cache Reserv
 
 Asset purges are free operations.
 
-Cache Reserve will also be purged along with edge cache when you send a purge by URL request.
+Cache Reserve will also be instantly purged along with edge cache when you send a purge by URL request.
```
Comment (tkornhammar) [inline src/content/docs/cache/advanced-configuration/cache-reserve.mdx]: Don't think we need the word "also" here.

File: src/content/docs/cache/index.mdx [hunk 3]
```diff
@@ -54,7 +54,7 @@ Use Cloudflare's persistent storage to increase cache times.
 
 <Feature header="Purge" href="/cache/how-to/purge-cache/">
 
-Clear cached files to force Cloudflare to fetch a fresh version of those files from your web server. You can purge files selectively or all at once.
+Instantly purge cached files to force Cloudflare to fetch a fresh versions from your web server files. You can purge specific files or all at once.
```
Comment (tkornhammar) [inline src/content/docs/cache/index.mdx]: Either remove the "a" in "a fresh" or make versions/files singular.

File: src/content/docs/cache/advanced-configuration/cache-reserve.mdx (hunk 2 above)
Comment (angelampcosta) [inline src/content/docs/cache/advanced-configuration/cache-reserve.mdx]:
```suggestion
Cache Reserve will be instantly purged along with edge cache when you send a purge by URL request.
```

File: src/content/docs/cache/how-to/purge-cache/purge-by-tags.mdx (hunk 1 above)
Comment (angelampcosta) [inline src/content/docs/cache/how-to/purge-cache/purge-by-tags.mdx]:
```suggestion
4. Use specific cache-tags to instantly purge your Cloudflare CDN cache of all content containing that cache-tag from your dashboard or [using our API](/api/operations/zone-purge).
```

File: src/content/docs/cache/index.mdx (hunk 3 above)
Comment (angelampcosta) [inline src/content/docs/cache/index.mdx]:
```suggestion
Instantly purge cached files to force Cloudflare to fetch fresh versions from your web server files. You can purge specific files or all at once.
```

**Review by marciocloudflare — APPROVED** [review]

File: src/content/docs/cache/how-to/purge-cache/index.mdx [hunk 4]
```diff
@@ -6,7 +6,7 @@ pcx_content_type: concept
 
 import { DirectoryListing, FeatureTable } from "~/components"
 
-Cloudflare offers different options to purge your content. Although single-file cache purging (purge by URL) is recommended, the following options are also available:
+Cloudflare's cache purge takes effect instantly, ensuring that any updates to your content are immediately reflected. Multiple options are available for purging content, with single-file cache purging (purge by URL) being the recommended method. However, the following additional options are also available:
```
Comment (tkornhammar) [inline src/content/docs/cache/how-to/purge-cache/index.mdx]: Suggestion: "Cloudflare's Instant Purge ensures that updates to your content are reflected immediately."

Comment (angelampcosta) [inline src/content/docs/cache/how-to/purge-cache/index.mdx]:
```suggestion
Cloudflare's Instant Purge ensures that updates to your content are reflected immediately. Multiple options are available for purging content, with single-file cache purging (purge by URL) being the recommended method. However, the following additional options are also available:
```

**Pull request merged**